presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1235,8 +1237,8 @@ private static boolean containsAllPrivilege(Set<PrivilegeGrantInfo> requestedPri
     private RetryDriver retry()
     {
         return RetryDriver.retry()
-                .exceptionMapper(exceptionMapper)
-                .stopOn(PrestoException.class);
+                .exponentialBackoff(thriftConfig.getMaxRetries(), thriftConfig.getMinBackoffDelay(), thriftConfig.getMaxBackoffDelay(), thriftConfig.getMaxRetryTime(),
+                        thriftConfig.getBackoffScaleFactor(), exceptionMapper, ImmutableList.of(), Optional.empty()).stopOn(PrestoException.class);
```

Comment:
Keep the `.stopOn()` call on a separate line for readability.

Reply:
@electrum  Done.